_sass/core/global.scss

```diff
@@ -117,6 +117,10 @@ p.byline {
   line-height: 1.8rem;
 }
 
+b,strong {
+  font-weight: bold;
+}
+
```

Comment:
Why does it need to be added? It sounds like this is probably just hiding some other styling which you hard-override.

Reply:
other elements has other `font-weight` set there
for example `p`:
```
p {
  font-size: 1rem;
  line-height: 1.5rem;
  font-weight: 100;
}
```
so I guess that other sub-elements that are used as children are inheriting this `font-weight`

Reply:
Hmm 🤔

Reply:
And also for the whote `body, html` is set the `font-weight`:

```
html,
body,
*
{
  font-size: 100%;
  font-weight: 100;
  font-family: "Nunito", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
}
```

let me try to remove the `*`, but I'm a bit afraid that it will break something 😄

Reply:
I don't know ... better not poke that :-o

Reply:
Yeah even without the wildcard it doesn't work 🤷